src/main/java/cn/nukkit/entity/passive/EntityChicken.java

```diff
@@ -38,7 +38,7 @@ public String getName() {
 
     @Override
     public Item[] getDrops() {
-        return new Item[]{Item.get(Item.RAW_CHICKEN), Item.get(Item.FEATHER)};
+        return new Item[]{Item.get(((this.isOnFire()==true) ? Item.COOKED_CHICKEN : Item.RAW_CHICKEN)), Item.get(Item.FEATHER)};
```

Comment:
isOnFire() already returns boolean

Reply:
I know, but its just the code style. Should I remove it?

Reply:
Yes, remove it.

Reply:
Done